crates/engine/tree/src/tree/mod.rs

```diff
@@ -2730,7 +2729,10 @@ where
     ) -> Result<(), InsertBlockErrorKind> {
         // Get the builder once, outside the thread
         let Some(state_provider_builder) = self.state_provider_builder(block.parent_hash())? else {
-            trace!(target: "engine::tree", parent=%block.parent_hash(), "Could not get state provider builder for prewarm");
```

Comment:
if unformattable macros / string literals go over the configured max rustfmt column width, it will just skip formatting the entire function